client/src/components/Comment/CommentComponent.jsx

```diff
@@ -1,33 +1,48 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/prop-types */
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Button, Comment, Form } from 'semantic-ui-react';
 import CommentReply from './Reply';
 import classes from './index.module.css';
+import useReplies from '../../hooks/useComments';
+import fromNow from '../../utils/date/fromNow';
 
 const CommentComponent = ({
-  image, author, postedOn, content,
+  commentId,
+   image,
+   author,
+   postedOn,
+  content,
+  blogId,
+  _id,
+  username,
 }) => {
+  const isTempComment = commentId.indexOf('temp') > -1;
```

Comment:
what is tempCpomment?

Reply:
Unless the comment is posted successfully in the backend. I need a temp Id to show adding and '...' for the posted on